### ISSUE TITLE
Date Not Clearing when state clears

### ISSUE THREAD

**Issue by kingeke** [jazeee/react-datetime-bootstrap#19]
So assuming the user submits a form and the state should reset, the selected date in the picker still remains.

A simple fix is to add this.datePicker.clear() in updateValue function

```
updateValue = (value) => {
		if (!this.datePicker) {
			return;
		}
		const {pickerOptions: {format}} = this.props;
		if (value !== undefined && value !== null) {
			if (moment.isMoment(value) || value.trim().length) {
				this.datePicker.date(value);
				if (!moment.isMoment(value)) {
					value = moment(value);
				}
				this.textInputElement.value = value.format(format);
				return;
			}
		}
		this.textInputElement && (this.textInputElement.value = "");
		this.datePicker.clear()
	}
```

**Comment by jazeee** [comment]
It does seem like you are right. I haven't looked at this code for quite a while, and it is getting a bit dated.
If you want to provide a pr, feel free to.

**Comment by kingeke** [comment]
sent

**Comment by jazeee** [comment]
I've merged #20 and published as v1.4.6 to NPM. (I did have to add a dev package - `babel-preset-es2015`, along with adding `yarn.lock`)
https://www.npmjs.com/package/react-datetime-bootstrap
Thanks much.

**Comment by kingeke** [comment]
Alright. thanks.
